server/server/Controllers/adminController.js

```diff
@@ -218,6 +220,22 @@ const removeOneOrManyRestaurants = async (req, res) => {
   }
 };
 
+
+// Admin retrieving all current orders with the status "delivering"
+const getDeliveringOrders = async (req, res) => {
+  try {
+     const deliveringOrders = await Order.find({ orderStatus: 'delivering' });
```

Comment:
This should be the same name of imported model
```suggestion
     const deliveringOrders = await ordersModel.find({ orderStatus: 'delivering' });
```